scripts/test-lucky13.py

```diff
@@ -117,6 +121,8 @@ def main():
             interface = arg
         elif opt == '-o':
             outdir = arg
+        elif opt == "--payload-len":
+            ciphertext_len = int(arg)
```

Comment:
Should we have some limitation for this variable? If we would go small enough or negative, the `payload_len` will end up negative, probably resulting in ugly tracebacks.

Reply:
while yes, it would be nice to provide a specific error message, the actual value depends on block size of the cipher and length of the MAC and the interaction between the two, so it's not a simple calculation

in practice, if you specify a too small length, you'll see an error like:
```
  File "/home/hkario/dev/tlsfuzzer/scripts/test-lucky13.py", line 238, in main
    ApplicationDataGenerator(bytearray(payload_len)),
ValueError: negative count
```
or
```
Error encountered while processing node <tlsfuzzer.messages.ApplicationDataGenerator object at 0x7fc82b760910> (child: <tlsfuzzer.expect.ExpectAlert object at 0x7fc82b760970>) with last message being: <tlslite.messages.ApplicationData object at 0x7fc82b6cc3d0>
Error while processing
Traceback (most recent call last):
  File "/home/hkario/dev/tlsfuzzer/scripts/test-lucky13.py", line 432, in main
    runner.run()
  File "/home/hkario/dev/tlsfuzzer/tlsfuzzer/runner.py", line 250, in run
    self.state.msg_sock.sendMessageBlocking(msg)
  File "/home/hkario/dev/tlsfuzzer/tlslite/messagesocket.py", line 186, in sendMessageBlocking
    for _ in self.sendMessage(msg):
  File "/home/hkario/dev/tlsfuzzer/tlslite/messagesocket.py", line 181, in sendMessage
    for res in self.flush():
  File "/home/hkario/dev/tlsfuzzer/tlslite/messagesocket.py", line 121, in flush
    for res in self.sendRecord(msg):
  File "/home/hkario/dev/tlsfuzzer/tlslite/recordlayer.py", line 625, in sendRecord
    data = self._macThenEncrypt(data, contentType)
  File "/home/hkario/dev/tlsfuzzer/tlslite/recordlayer.py", line 486, in _macThenEncrypt
    data = self.addPadding(data)
  File "/home/hkario/dev/tlsfuzzer/tlsfuzzer/messages.py", line 1909, in new_add_padding
    raise ValueError("min_length set too "
ValueError: min_length set too high for message: 257
```

which may not be perfect, but they do point to length being the problem

also, if you are specifically interested in behaviour of small padding, then you can always run just specific tests with very small payload length, and those will work fine...